libshaderc/src/shaderc_test.cc

```diff
@@ -52,6 +52,14 @@ TEST(Init, MultipleThreadsCalling) {
   shaderc_compiler_release(compiler3);
 }
 
+TEST(Init, SPVVersion) {
+  unsigned int version = 0;
+  unsigned int revision = 0;
+  shaderc_get_spv_version(&version, &revision);
+  EXPECT_EQ(version, spv::Version);
```

Comment:
Looks like you got the arguments swapped -- spv::Version is the expected, version the actual.